src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -502,9 +502,15 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
 
                 case 2:
                 {
-                    assert(instrIsRMW);
-
-                    if (intrin.op3->IsVectorZero())
+                    if (!instrIsRMW)
+                    {
+                        assert(intrin.op3->IsVectorZero());
```

Comment:
`intrin.op3` should be `nullptr` because there are just 2 operands involved.

Reply:
In this context `intrin` is `Sve_ConditionalSelect`, not `Sve_AbsoluteCompare*`.